### PR TITLE
REP-3294-Gather-production-data

### PR DESCRIPTION
**Pull request by LucasVelvis**
[Optionally] Jira ticket link: https://repowerednl.atlassian.net/browse/REP-3294

### CODE REVIEW

File: src/marloes/data/profiles/Amen-LG-10-EW-Mounted.parquet
Comment (Matthiasd23) [inline src/marloes/data/profiles/Amen-LG-10-EW-Mounted.parquet:1]: Rename to what type of production profile? We don't need (or want) names i'd say.

File: pyproject.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ license = "MIT"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.12"
+python = "^3.11"
```
Comment (Matthiasd23) [inline pyproject.toml:10]: All right then...

File: src/marloes/hub/env.py [hunk 2]
```diff
@@ -7,10 +7,13 @@
 from agents.solar import SolarAgent
 from agents.wind import WindAgent
 
+
+# TODO: Maybe change this to EnergyValley?
```
Comment (Matthiasd23) [inline src/marloes/hub/env.py:11]: Sure, this would be fine and more Reformers like

**Review by Matthiasd23 — APPROVED** [review]

**Comment by Matthiasd23** [comment]
Addition to review: Was it not part of the ticket to have the agent import the data as well?

**Comment by LucasVelvis** [comment]
> Addition to review: Was it not part of the ticket to have the agent import the data as well?

No I think i. have another ticket for that

**Comment by Matthiasd23** [comment]
LGTM, crazy kW/kWh things

**Pull request merged**